data/stylesheet/_index.scss

```diff
@@ -5,33 +5,50 @@ $fg-color: $BLACK_500;
 
 $text_color: $fg_color;
 
-.roundy-label,
-.roundy-label:hover,
-.roundy-label:selected,
-.roundy-label:selected:focus,
-.roundy-label:hover:selected {
-    background-image: none;
-    box-shadow: none;
-    color: rgba($text_color, 0.7);
-    font-weight: 700;
-    border-radius: 10px;
-    padding: 0 6px;
+roundy-label,
+roundy-label:hover,
+roundy-label:selected,
+roundy-label:selected:focus,
+roundy-label:hover:selected {
     margin: 0 3px;
-    border-width: 0;
 
-    @if $color-scheme == "light" {
-        background-color: $SILVER_300;
-    } @else if $color-scheme == "dark" {
-        background-color: $SILVER_900;
+    .small-label {
+        padding: 0;
     }
-}
 
-.green {
-    background-color: rgba($LIME_300, 0.6);
-}
+    .value {
+        color: #{'alpha(@text_color, 0.7)'};
+        font-feature-settings: "tnum";
+        font-weight: 700;
+        border-radius: 10px;
+        padding: 0 6px;
+
+        @if $color-scheme == "light" {
+            background-color: $SILVER_300;
+        } @else if $color-scheme == "dark" {
```

Comment:
Looks like we have removed in-app color-scheme check and text in lables for a dark mode is too dark now. How to reintroduce it in GTK4?
<img width="1353" height="738" alt="image" src="https://github.com/user-attachments/assets/eeca51a5-67e1-4048-bbbc-202da9b1e9a9" />

Reply:
Did you run `ninja-clean` first? Here's how it looks here:

<img width="1018" height="781" alt="Screenshot from 2025-07-14 14 03 15" src="https://github.com/user-attachments/assets/1c9963e6-23cd-4d3c-83fe-cd733cadc98a" />

Reply:
Ah, yes. My bad.